src/main/resources/mediator-registration-info.json

```diff
@@ -5,10 +5,17 @@
   "description": "A mediator for handling system integration between Epicor and THSCP",
   "endpoints": [
     {
-      "name": "THSCP-Mediator-Epicor Route",
+      "name": "Health Commodities Funding Route",
```

Comment:
You'll need to update the endpoints and the default channel configurations to match the created routes in the [Mediator main routing table](https://github.com/SoftmedTanzania/thscp-mediator-epicor/blob/ff1d9af01418b3f62bf9f134198ac5d745879313/src/main/java/tz/go/moh/him/thscp/mediator/epicor/MediatorMain.java#L27)